### ISSUE TITLE
Cannot open Jupyter R-notebook in a project subfolder inside emacs 

### ISSUE THREAD

**Issue by Krisselack** [millejoh/emacs-ipython-notebook#410]
## Check list

- [x] Read the "Avoiding Common Emacs Traps" section in
  https://github.com/millejoh/emacs-ipython-notebook/blob/master/CONTRIBUTING.md
- [x] Does IPython works from the web browser? 
- [x] Confirm the problem was not due to badly compiled
  files by removing all the `*.elc` files from source directory of EIN and
  its dependencies.
- [x] Confirm that the libraries are loaded as expected and are at the required version.
  (You can check the location in the "System info" section below)

## Description of the problem you have

Using R kernel; wanted to reach subfolder of project in emacs via notebooklist. The only content is the ipynb file, a hidden checkpoint-folder and a few images. 
After opening it in emacs ein:notebooklist, I get the following error: 

```cl
[W 20:12:44.980 NotebookApp] 404 GET /api/contents/tutorialLesson_3 (127.0.0.1): No such file or directory: tutorialLesson_3
[W 20:12:44.980 NotebookApp] No such file or directory: tutorialLesson_3
[W 20:12:44.980 NotebookApp] 404 GET /api/contents/tutorialLesson_3 (127.0.0.1) 0.98ms referer=None
[W 20:12:48.278 NotebookApp] 404 GET /api/contents/tutorialLesson_2 (127.0.0.1): No such file or directory: tutorialLesson_2
[W 20:12:48.279 NotebookApp] No such file or directory: tutorialLesson_2
[W 20:12:48.279 NotebookApp] 404 GET /api/contents/tutorialLesson_2 (127.0.0.1) 0.98ms referer=None
[W 20:12:51.414 NotebookApp] 404 GET /api/contents/tutorialLesson_1 (127.0.0.1): No such file or directory: tutorialLesson_1
[W 20:12:51.414 NotebookApp] No such file or directory: tutorialLesson_1
[W 20:12:51.415 NotebookApp] 404 GET /api/contents/tutorialLesson_1 (127.0.0.1) 1.01ms referer=None
[W 20:13:44.902 NotebookApp] 404 GET /api/contents/tutorialLesson_2 (127.0.0.1): No such file or directory: tutorialLesson_2
[W 20:13:44.903 NotebookApp] No such file or directory: tutorialLesson_2
[W 20:13:44.903 NotebookApp] 404 GET /api/contents/tutorialLesson_2 (127.0.0.1) 0.99ms referer=None
```

Status line: 
```20:13:44:921: [debug] ein:query-contents--complete STATUS: 404 DATA: (:message No such file or directory: tutorialLesson_2 :reason nil) @#<buffer *ein:notebooklist http://127.0.0.1:8888*>```

I suspect that a slash for the subfolder is missing. 

## Steps to reproduce the problem

1. open jupyter via emacs 
2. enter notebboklist 
3. enter subfolder 

## Expected output
Opened notebook. 

## Your EIN configuration (in .emacs.d/init.el or somewhere else)
(package-initialize)
(require 'ein)
(require 'ein-loaddefs)
(require 'ein-notebook)
(require 'ein-subpackages)

## Your IPython configuration

1. What is your IPython version? (run `ipython --version`):
6.4.0 

2. How do you start IPython? (e.g., `ipython notebook --port 9999`):
ein:jupyter-server-start

3. What is your IPython notebook port number or URL?: 
http://localhost:8888/tree

## Additional information (if any)
Folder permission was checked for. Fedora 29 64 bit. 
Locale: 
```cl
LANG=en_US.UTF-8
LC_CTYPE="en_US.UTF-8"
LC_NUMERIC=de_DE.UTF-8
LC_TIME=de_DE.UTF-8
LC_COLLATE="en_US.UTF-8"
LC_MONETARY=de_DE.UTF-8
LC_MESSAGES="en_US.UTF-8"
LC_PAPER=de_DE.UTF-8
LC_NAME="en_US.UTF-8"
LC_ADDRESS="en_US.UTF-8"
LC_TELEPHONE="en_US.UTF-8"
LC_MEASUREMENT=de_DE.UTF-8
LC_IDENTIFICATION="en_US.UTF-8"
LC_ALL=
```

## System info:

```cl
("EIN system info"
 :emacs-version
 "GNU Emacs 26.1 (build 1, x86_64-redhat-linux-gnu, GTK+ Version 3.23.2)
 of 2018-08-13"
 :emacs-bzr-version nil
 :window-system x
 :emacs-variant nil
 :os
 (:uname
  "Linux merlin 4.19.2-301.fc29.x86_64 #1 SMP Sat Nov 17 17:58:01 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux
"
  :lsb-release nil)
 :image-types
 (svg imagemagick png gif tiff jpeg xpm xbm pbm postscript)
 :image-types-available
 (svg imagemagick png gif tiff jpeg xpm xbm pbm postscript)
 :request
 (:backend curl)
 :ein
 (:version "0.15.0"
           :source-dir "/home/christian/.emacs.d/elpa/ein-20181015.2138/")
 :lib
 ((:name "websocket"
         :path "~/.emacs.d/elpa/websocket-20180423.16/websocket.elc"
         :featurep t
         :version-var websocket-version
         :version "1.9")
  (:name "request"
         :path "~/.emacs.d/elpa/request-20170201.147/request.elc"
         :featurep t
         :version-var request-version
         :version "0.3.0")
  (:name "auto-complete"
         :path "~/.emacs.d/elpa/auto-complete-20170125.245/auto-complete.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "auto-complete"
         :path "~/.emacs.d/elpa/auto-complete-20170125.245/auto-complete.elc"
         :featurep t
         :version-var nil
         :version nil)
  (:name "popup"
         :path "~/.emacs.d/elpa/popup-20160709.729/popup.elc"
         :featurep t
         :version-var popup-version
         :version "0.5.3")
  (:name "python"
         :path "/usr/share/emacs/26.1/lisp/progmodes/python.elc"
         :featurep t
         :version-var nil
         :version nil)))
```

**Comment by dickmao** [comment]
https://github.com/millejoh/emacs-ipython-notebook/issues/364

Your version indicates 10.15 and this bug (mine) was fixed 10.26.

Please try to `package-install` a version later than 10.26.

**Comment by Krisselack** [comment]
Ah thanks. Updated and now it works.
